packages/breadboard-ui/src/elements/editor/editor.ts

```diff
@@ -327,9 +327,9 @@ export class Editor extends LitElement {
       return;
     }
 
-    const nextNodeId =
-      (this.loadInfo?.graphDescriptor?.nodes.length || 1_000) + 1;
-    const id = `${data}-${nextNodeId}`;
+    const nextNodeId = globalThis.crypto.getRandomValues(new Uint8Array(3));
```

Comment:
crypto.randomUUID and then take the last 6 bytes, if you want to keep it short but unique...

Reply:
Yeah, I was going to try and avoid `randomUUID` because it's so long, but you're right I can just take a segment of it instead. Ultimately I need to check for clashes anyway, but this is an improvement on what we had.